lib/controllers/authorization-controller/questions.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
+  CONFIRM_ALLOW_BROWSER_SIGN_IN: [
+    {
+      message: "Do you confirm using the browser to sign in to Alexa Skills Kit Tools?",
```

Comment:
There isn't a clear indicator for the input of this prompt, and the grammar is slightly off. A couple of alternative suggestions:

'Do you confirm that you used the browser to sign in to Alexa Skills Kit Tools? [y/n]: '

'Press enter to continue when you have completed signing in with the browser... '